ksql-engine/src/main/java/io/confluent/ksql/ddl/commands/RegisterTopicCommand.java

```diff
@@ -117,4 +118,25 @@ public DdlCommandResult run(final MetaStore metaStore) {
 
     return new DdlCommandResult(true, "Topic registered");
   }
+
+  private String getSourceType(final MetaStore metaStore) {
+    final StructuredDataSource source = metaStore.getSource(topicName);
+    if (source == null) {
+      return "An entity";
```

Comment:
This can be `A topic` right?  As `metaStore.getTopic(topicName)` above has returned non-null to get to this point in the code, so we know its a topic.  (You'll notice your example output currently never says `A topic` because that branch of your code may not be reachable).

Reply:
Also, could you add tests to `RegisterTopicCommandTest.java` to cover these different paths?

I know you've added tests to the CreateStream and CreateTable test files, but this doesn't cover all code paths, and really each path should be covered by _this_ classes test file.